### PR TITLE
found the missing part and corrected it.

### PR DESCRIPTION
**Pull request by goodchai0**
Now the search is visible.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export default function Home() {
 
       
 
-<div className="w-full md:w-1/2 flex flex-col items-center h-64 mx-auto ">
+<div className="w-full md:w-1/2 flex flex-col items-center h-64 mx-auto" style={{color:"black"}}>
```
Comment (AnkitaMalik22) [inline pages/index.js:48]: <div className="w-full md:w-1/2 flex flex-col items-center h-64 mx-auto text-black">

**Review by AnkitaMalik22 — CHANGES_REQUESTED** [review]
you can just add a " text-black " class to it

File: pages/index.js (hunk 1 above)
Comment (AnkitaMalik22) [inline pages/index.js:48]:
```suggestion
<div className="w-full md:w-1/2 flex flex-col items-center h-64 mx-auto text-black">
```

**Review by AnkitaMalik22 — CHANGES_REQUESTED** [review]

File: pages/index.js (hunk 1 above)
Comment (AnkitaMalik22) [reply pages/index.js:48]: you can just add a text-black class to it

**Comment by goodchai0** [comment]
done! I have updated the code.

**Review by AnkitaMalik22 — APPROVED** [review]

**Comment by AnkitaMalik22** [comment]
Thanks for contributing!

**Comment by goodchai0** [comment]
Thank You so much for merging.

**Pull request merged**